index.d.ts

```diff
@@ -1,4731 +1,8 @@
 // DO NOT EDIT THIS FILE DIRECTLY
 // make edits in scripts/generate-types.js
 
-import http = require("http");
+import { EventNames } from "./lib/generated/event-names";
+import { EventPayloads } from "./lib/generated/event-payloads";
+import { Webhooks } from "./lib/generated/api";
```

Comment:
Instead of importing and then exporting, you can just do the following

```suggestion
export { EventNames } from "./lib/generated/event-names";
export { EventPayloads } from "./lib/generated/event-payloads";
export { Webhooks } from "./lib/generated/api";
```